packages/cli/src/helpers/default-flag-args.ts

```diff
@@ -71,5 +71,7 @@ export const defaultFlagArgs: ITBConfig = {
   runtimeStats: 'false',
   emulateDevice: null,
   routes: ['/'],
-  regressionThreshold: '0ms',
+  inputFilePath: './tracerbench-results/compare.json',
+  outputFilePath: './tracerbench-results/artifact/',
```

Comment:
lets not give an option for outputFilePath. we allow the configuration folder for all tracerbench-results output via this config and then just dump all things tracerbench into that.

Reply:
we can check if the pdf artifact results exists and if they do then append a version number to the pdf file. eg "./tracerbench-results/artifact-1.pdf"

Reply:
Wait @lynchbomb I checked compare's command's configurations and noticed that "tbResultsFile" flag is customize-able by the user. Should be mimic same behavior?

Reply:
> we allow the configuration folder for all tracerbench-results output via this config and then just dump all things tracerbench into that.

^^^ yes this is `tbResultsFile`. This is the folder the user can specify to dump everything from tracerbench into